resources/dicts/events/death/beach.json

```diff
@@ -58,16 +57,16 @@
         ],
         "tags": [
             "shock",
-            "no_body"
+            "no_body",
+            "all_lives"
         ],
         "weight": 20,
         "event_text": "m_c was lost to the sea while trying to save r_c from drowning.",
         "m_c": {
-            "age": [],
-            "status": [],
+            "age": [ "any" ],
```

Comment:
should probably restrict this so that kittens and newborns are trying to save someone